app/views/content/index.html.erb

```diff
@@ -134,7 +134,14 @@
               </td>
               <td class="govuk-table__cell govuk-table__cell--shaded" data-gtm-id="document_type-column"><%= item.document_type %></td>
               <td class="govuk-table__cell govuk-table__cell--numeric" data-gtm-id="upviews-column"><%= item.upviews %></td>
-              <td class="govuk-table__cell govuk-table__cell--shaded" data-gtm-id="satisfaction-column"><%= item.user_satisfaction %></td>
+              <td class="govuk-table__cell govuk-table__cell--shaded" data-gtm-id="satisfaction-column">
+                <%= item.user_satisfaction_percentage %>
+                <% if item.user_satisfaction_responses %>
+                  <span class="satisfaction-column__responses">
+                    (<%= item.user_satisfaction_responses %>)
+                  </span>
+                <% end %>
+              </td>
```

Comment:
Could we re-write the logic like so that we know that `user_satisfaction_percentage` is always a percentage. Currently it can return `No responses` which could be confusing.  
```erb
<% if item.user_satisfaction_percentage %>
    <%= item.user_satisfaction_percentage %>
    <span class="satisfaction-column__responses"> (<%= item.user_satisfaction_responses %>) </span>
<% else %>
   <%= item.user_satisfaction_responses %>
<% end %>
```